### DISCUSSION TITLE
What do they mean?

### DISCUSSION THREAD

**Discussion by Pridanc** [mriscoc/Ender3V2S1#983] [General]
First off, Miguel thanks much for your outstanding work (and all who help). I'm new to all this but saw that what you had done with Marlin works so well for folks like me. I'm one of those who don't want to know how a tool (Ender 3 v2) is made to run inside and out, but do want to be able to use it in a more complete fashion. Making the changes you have allow me to use the UI that allows folks like me to jump right in and use the tool more successfully. 
 
Second: I'll imagine that what I'm about to ask is easily found but lacking the words to search deep enough to get a meaningful answer, I'm holding my hand up asking if you (anyone here) can steer me to where I can find descriptions of what several of the lines in the Enders UI mean and or do? I've looked on Marlin's site but the explanations are either not there or too simple for an ignorant old fool like me to grasp. For example Tilting Grid. (and its off shoots) and Smart fill in? Even things under Probe Settings like Reset. What does that do? I'm also chicken to randomly click things as the printer works dang well now and I'd hate to mess it up without knowing why.. OK, yes, I can "go back" as I do keep decent records but still.

Third: After updating my MRiscoC firmware and prior to a test print (but I'd loaded the print) I went back to verify that all the old settings that I'd recorded prior to the upgrade where as I expected them to be. To my surprise many had changed. Ignorantly I'll ask if some of the settings don't change with each print based on what the slicer has requested?  I've never looked at all the settings after loading a print before so I've NO clue. The obvious ones, temps, fans etc, yes I look but some of the other, never.  And I only looked this time because I'd done the update and needed to make sure that E-steps etc were as they had been.

OK that's it! Can't thank everyone enough for this journey as I've loved every minute of it.
Sincerely, PC

**Comment by mriscoc** [comment]
Hi PC, indeed the firmware has some advanced settings that may confuse beginners, I suggest you only change things that you previously had information and not touch parameters randomly.

Near all menu options have a corresponding Marlin G-code, you can get a lot of information here: https://marlinfw.org/meta/gcode/

Many configurations parameters can be changed by menu and also by G-code, so, if you set some parameter manually by menu but also your g-code file change that parameter, you will see the latest change in the screen.
